VoiceAttackResponder/VoiceAttackVariables.cs

```diff
@@ -719,10 +760,6 @@ protected static void setShipyardValues(List<Ship> shipyard, ref dynamic vaProxy
 
         private static void setStarSystemValues(StarSystem system, string prefix, ref dynamic vaProxy)
         {
-            if (system == null)
-            {
-                return;
-            }
```

Comment:
Why not keep this null check?

Reply:
I removed it because if the system object IS null, you want those VA variables set to '`Not Set`. Otherwise they will remain set to the previous, no longer valid, value.